### DISCUSSION TITLE
Runout sensor

### DISCUSSION THREAD

**Discussion by fpmirabile** [Jyers/Marlin#969] [Ideas]
Hey, I am having this issue, wanted to post as an "idea" while I am learning how to build marlin 😓 

I have a filament [sensor](https://salesoftheday.com/wp-content/uploads/2019/04/KINGROON-3D-Printer-Filament-Break-Detection-Module-With-1M-Cable-Run-out-Sensor-Material-Runout-Detector.jpg) and seems need to be reverted (M119 says open when has filament), so I am struggling myself on each print.
Is it possible to add this in the UI?

**Comment by smooreace** [comment]
Compile it with the correct setting
Prosper

**Comment by gothcha** [comment]
This has been discussed a number of times, no it is not possible to change the polarity of the sensor from within the UI, it can only be defined at compilation unfortunately.
